library/src/lib/combobox/combobox.component.ts

```diff
@@ -115,10 +122,12 @@ export class ComboboxComponent implements ControlValueAccessor, OnInit, OnChange
     inputTextValue: string;
 
     /** @hidden */
-    onChange: any = () => {};
+    onChange: any = () => {
+    };
```

Comment:
Is it our way to display empty functions? I guess it was made automatically by lint.

Reply:
yep automatic change, I'll revert